src/backend/core/tests/test_usage_metrics_scraping.py

```
@@ -0,0 +1,451 @@
+from urllib.parse import parse_qs, urlparse
+
+from django.db.models import Count
+
+import pytest
+
+from core import factories, models
+from core.tasks.metrics import (
+    fetch_metrics_from_service,
+    fetch_usage_metrics_from_service,
+    store_service_metrics,
+    store_service_metrics,
```

Ruff (0.14.3)

Redefinition of unused `store_service_metrics` from line 21

Remove definition: `store_service_metrics`

(F811)